src/Components/MenuDirectories.module.css

```diff
@@ -0,0 +1,25 @@
+nav{
+    width: 50%;
+    max-width: 300px;
+    height: 50px;
+}
+nav > ul{
+    display:flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    align-items: center;
+    padding-inline-start: 0;
+    margin-block: 0;
+    height: 100%;
+}
+li:hover{
```

Comment:
we probably want to use more specific names using className=... instead of just "li" or "ul"